mne/decoding/receptive_field.py

```diff
@@ -311,12 +367,15 @@ def _delay_time_series(X, tmin, tmax, sfreq, newaxis=0, axis=0):
     newaxis : int
         The axis in the output array that corresponds to time delays.
         Defaults to 0, for the first axis.
+    fill_mean : bool
```

Comment:
If True...the fill value will be the mean along the time dimension of the feature. If False, the fill value will be zero.